tests/resources/aria-at-harness.mjs

```diff
@@ -162,10 +176,107 @@ function executeScriptInTestPage() {
   }
 }
 
+/**
+ * @typedef AT
+ * @property {string} name
+ * @property {string} key
+ */
+
+/**
+ * @typedef Support
+ * @property {AT[]} ats
+ * @property {{system: string[]}} applies_to
+ * @property {{directory: string, name: string}[]} examples
+ */
+
+/**
+ * @typedef {{[mode in ATMode]: {[assitiveTech: string]: string;};}} CommandsAPI_ModeInstructions
```

Comment:
Acknowledging that `assistiveTech` does sounds kind of cool, given that `at` is used throughout the project, including the repo name, I think just `at` (or maybe `atName` based on the context?) would be a better term to use.

Reply:
`atName` sounds good.